meinberlin/apps/budgeting/api.py

```diff
@@ -185,6 +217,14 @@ class ProposalViewSet(ModuleMixin,
                        'dailyrandom',)
     search_fields = ('name', 'ref_number')
 
+    @property
+    def ordering(self):
+        if module_is_between_phases('meinberlin_budgeting:support',
+                                    'meinberlin_budgeting:voting',
+                                    self.module):
+            return ['-positive_rating_count']
+        return ['dailyrandom']
+
```

Comment:
this is adding the default ordering, as it is dynamic in our case, had to do it as property